File: RepoTransBench/target_projects/Java/Go/woodpecker-framework_ysoserial-for-woodpecker/tests/original/yso_config_smoke_test.go
package original

import (
	"testing"
	"github.com/stretchr/testify/assert"
	yso "woodpecker-framework-ysoserial"
)

func TestYsoConfigSmoke_DefaultConfig(t *testing.T) {
	conf := yso.NewYsoConfig()
	assert.NotNil(t, conf)
	assert.NotNil(t, conf.GetConfig())
}

func TestYsoConfigSmoke_SetAndGetConfig(t *testing.T) {
	conf := yso.NewYsoConfig()
	props := make(map[string]string)
	props["foo"] = "bar"
	conf.SetConfig(props)
	assert.Equal(t, "bar", conf.GetConfig()["foo"])
}

func TestYsoConfigSmoke_ToString(t *testing.T) {
	conf := yso.NewYsoConfig()
	conf.GetConfig()["k"] = "v"
	str := conf.String()
	assert.Contains(t, str, "k")
	assert.Contains(t, str, "v")
}

File: RepoTransBench/target_projects/Java/C#/woodpecker-framework_ysoserial-for-woodpecker/tests/original/YsoConfigSmokeTest.cs
using System;
using Xunit;

namespace WoodpeckerYsoserial.Tests
{
    public class YsoConfigSmokeTest
    {
        [Fact]
        public void TestDefaultConfig()
        {
            var conf = new YsoConfig();
            Assert.NotNull(conf);
            Assert.NotNull(conf.Config);
        }

        [Fact]
        public void TestSetAndGetConfig()
        {
            var conf = new YsoConfig();
            var props = new System.Collections.Specialized.NameValueCollection();
            props.Set("foo", "bar");
            conf.Config = props;
            Assert.Equal("bar", conf.Config["foo"]);
        }

        [Fact]
        public void TestToString()
        {
            var conf = new YsoConfig();
            conf.Config.Set("k", "v");
            Assert.Contains("k", conf.ToString());
            Assert.Contains("v", conf.ToString());
        }
    }
}